packages/dev/serializers/src/glTF/2.0/Extensions/KHR_lights_punctual.ts

```diff
@@ -59,132 +69,113 @@ export class KHR_lights_punctual implements IGLTFExporterExtensionV2 {
-                    };
-
-                    // Avoid duplicating the Light's parent node if possible.
-                    const parentBabylonNode = babylonNode.parent;
-                    if (parentBabylonNode && parentBabylonNode.getChildren().length == 1) {
-                        const parentNode = this._exporter._nodes[nodeMap[parentBabylonNode.uniqueId]];
-                        if (parentNode) {
-                            const parentTranslation = Vector3.FromArrayToRef(parentNode.translation || [0, 0, 0], 0, TmpVectors.Vector3[0]);
-                            const parentRotation = Quaternion.FromArrayToRef(parentNode.rotation || [0, 0, 0, 1], 0, TmpVectors.Quaternion[0]);
-                            const parentScale = Vector3.FromArrayToRef(parentNode.scale || [1, 1, 1], 0, TmpVectors.Vector3[1]);
-                            const parentMatrix = Matrix.ComposeToRef(parentScale, parentRotation, parentTranslation, TmpVectors.Matrix[0]);
-
-                            const translation = Vector3.FromArrayToRef(node.translation || [0, 0, 0], 0, TmpVectors.Vector3[2]);
-                            const rotation = Quaternion.FromArrayToRef(node.rotation || [0, 0, 0, 1], 0, TmpVectors.Quaternion[1]);
-                            const matrix = Matrix.ComposeToRef(Vector3.OneReadOnly, rotation, translation, TmpVectors.Matrix[1]);
-
-                            parentMatrix.multiplyToRef(matrix, matrix);
-                            matrix.decompose(parentScale, parentRotation, parentTranslation);
-
-                            if (parentTranslation.equalsToFloats(0, 0, 0)) {
-                                delete parentNode.translation;
-                            } else {
-                                parentNode.translation = parentTranslation.asArray();
-                            }
-
-                            if (Quaternion.IsIdentity(parentRotation)) {
-                                delete parentNode.rotation;
-                            } else {
-                                parentNode.rotation = parentRotation.asArray();
-                            }
-
-                            if (parentScale.equalsToFloats(1, 1, 1)) {
-                                delete parentNode.scale;
-                            } else {
-                                parentNode.scale = parentScale.asArray();
-                            }
-
-                            parentNode.extensions ||= {};
-                            parentNode.extensions[NAME] = lightReference;
-
-                            // Do not export the original node
-                            resolve(null);
-                            return;
-                        }
-                    }
-
-                    node.extensions ||= {};
-                    node.extensions[NAME] = lightReference;
+            // If node was nullified (marked as skippable) earlier in the pipeline, or it's not a light, skip

```

Comment:
I had left this comment for myself, since I also wasn't sure why `postExportNodeAsync` accepted a `Nullable<INode>`. I incorrectly thought that a null node meant it was skipped... 

Regardless, I see that it's impossible for node to be null. Actually, It's impossible for any of the glTF objects to be null when passed into these extension methods. 

I've gone ahead and removed all of the relevant Nullable types. Done.